File: Programmers/Inho/Level01/부족한 금액 계산하기.cs
using System;

class Solution
{
    public long solution(int price, int money, int count)
    {
        long pay = 0;
        for (int i = 0; i < count; i++)
        {
            pay += (long)price * (i + 1);
        }
        if (pay < (long)money)
            return 0;
        return pay - (long)money;

    }
}

File: Programmers/Inho/C++/Level01/부족한 금액 계산하기.cpp
using namespace std;

long long solution(int price, int money, int count)
{
    long long answer = -1;
    long long nowMoney = 0;
    // 현재 돈에서 프라이스 까고, 남은거 음수치면 될듯
    for(int i = 0;i<count;i++){
        nowMoney += price*(i+1);
    }
    answer = nowMoney - money;
    if(money>=nowMoney)
        return 0;
    return answer;
}